### DISCUSSION TITLE
How to change the website direction from ltr to rtl for nextra BLOG

### DISCUSSION THREAD

**Discussion by TajAlasfiyaa** [shuding/nextra#2504] [Q&A]
i want to make blog for myself but in my original language but its direction is rtl .
i try to add dir:'rtl' and direction:'rtl' but none of this work in my blog.

**Comment by ahmedelq** [comment]
In your `next.config.js`, add: 
```javascript
module.exports = withNextra({
  i18n: {
    locales: [..., 'ar-SA'],
  }
})
````

and in your `theme.config.jsx` add:
```javascript
i18n: [
....
  { locale: 'ar-SA', text: 'العربية', direction: 'rtl' }
]
```
